packages/build-config/configs/node.js

```diff
@@ -19,6 +21,12 @@ function findNodeModules (start) {
   return modulesDir;
 }
 
+function shouldIncludeExternalModuleInBundle (module) {
```

Comment:
I tried to stick to current naming conventions: https://github.com/xing/hops/blob/master/packages/build-config/lib/check-esnext.js#L30 also: We don't use `module` here, so I don't think shadowing it would be a problem, but would also be willing to rename it.